src/FilterButton.js

```diff
@@ -0,0 +1,47 @@
+import Button from '@material-ui/core/Button';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import React, { Component } from 'react';
+
+export default class FilterButton extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { popoverOpen: false };
+  }
+
+  openPopover(event) {
+    event.preventDefault(); // Prevent ghost click.
+
+    this.setState({
+      popoverOpen: true,
+      anchorEl: event.currentTarget,
+    });
+  };
+
+  closePopover() {
+    this.setState({ popoverOpen: false });
+  };
+
+  render() {
+    return (
+      <center style={this.props.style}>
+        <Button
```

Comment:
I think a `<Select>` component would be more appropriate for this UI. It will keep the selected filter type visible so it's clear what's currently shown.

Demo: https://material-ui.com/demos/selects/

Reply:
Oh nice agreed!